src/history/hs_cursor.c

```diff
@@ -9,11 +9,11 @@
 #include "wt_internal.h"
 
 /*
- * __wt_hs_row_search --
+ * __wti_hs_row_search --
```

Comment:
If there is only one place where the `\_\_wti_` prefix is relevant, I don't think we should introduce this new protocol, as we have different solutions to the problem (as touched on in the ticket)

Reply:
I agree with you on this one. Most of the function are called from both within the history store subsystem and outside the subsystem and that's why we use `__wt_` prefix. I am not sure how this situation is for other sub-systems, but definitely a single intra-module function in history store module doesn't warrant a new naming prefix.